lib/languages/js/source-code/source-code.js

```diff
@@ -1038,27 +983,30 @@ class SourceCode extends TokenStore {
         const directives = [];
 
         this.getInlineConfigNodes().forEach(comment => {
-            const { directivePart, justificationPart } = commentParser.extractDirectiveComment(comment.value);
 
-            // Step 1: Extract the directive text
-            const match = directivesPattern.exec(directivePart);
+            // Step 1: Parse the directive
+            const {
+                label,
+                value,
+                justification: justificationPart
+            } = commentParser.parseDirective(comment.value);
+
+            const match = directivesPattern.exec(label);
 
             if (!match) {
                 return;
             }
```

Comment:
```suggestion
```

I believe there's no need for this as `this.getInlineConfigNodes()` already returns only those that pass this check.